www_src/components/select/select.less

```diff
@@ -0,0 +1,12 @@
+.select {
+  -webkit-appearance: none;
```

Comment:
oh weird. do we control the "list of properties to prefix"? (that feels like a thing we should be able to control"